### PR TITLE
Update community rules and guidelines

### PR DESCRIPTION
**Pull request by mikekaminsky**

### CODE REVIEW

File: content/community.md [hunk 1]
```diff
@@ -18,11 +18,32 @@ Once you have received your invite, please do the following:
...
 * Add a profile photo
 * Introduce yourself in the [#introductions](https://locallyoptimistic.slack.com/messages/CBB4QCX4N) channel
 
 In Slack, remember that channels are free. If you want to start a conversation on a topic that doesn't have a channel dedicated to it, you can create a channel and invite others to join you.
 
+###  Participating in the Slack community
+
+In general, in this community, we trust that everyone will behave in ways that they believe are appropriate. There are some hard-and-fast rules in our [code of conduct]({{< relref "code_of_conduct.md" >}}), which we expect everyone to adhere to: specifically, rules prohibiting harassment and intolerant behavior. However, in general, we prefer to have guidelines for behavior rather than hard-and-fast rules in a way that promotes organic growth and discussion. 
+
+We have had some great discussion in the #communityrules channel about how community members who also work for vendors should behave with respect to their and their competitor's competitors' products. There are three concerns we feel the need to address explicitly:
```
Comment (ilanman) [inline content/community.md]: `competitor's competitors'`

File: content/community.md [hunk 2]
```diff
@@ -18,11 +18,32 @@ Once you have received your invite, please do the following:
...
+
+* It should be clear when someone is making a comment if they are potentially biased
+* We do not want vendors bad-mouthing each others other's products
```
Comment (ilanman) [inline content/community.md]: `others other's`

File: content/community.md [hunk 3]
```diff
@@ -18,11 +18,32 @@ Once you have received your invite, please do the following:
...
+* We do not want this slack to become a sales-channel for vendors where conversations are consistently dominated by people promoting their own products.
```
Comment (ilanman) [inline content/community.md]: Just a thought - do we want to change the words from "We" to "Our community"? Gives it a more inclusive, less "up from on high" feels. Of course, "We" could mean "Our community"...

**Comment by mikekaminsky** [comment]
@ilanman thanks! I added some parentheticals to make it clear that "we" refers to the community.

File: content/community.md [hunk 4]
```diff
@@ -17,12 +17,33 @@ Once you have received your invite, please do the following:
...
+
+In order to address these, we (the community) want to propose two rules and two guidelines
+
+* Rule: if you are a vendor or an employee of a vendor-like entity, you should note that affiliation in your slack profile so that those reading the commentary can understand your potentially biased position
+* Guideline: make sure that we always keep the tone civil — in general, vendors should avoid commenting on a competitors’ technology unless they have specific nuances and differences they want to discuss. Simply saying “x is bad” would violate this guideline
+  * Note that this does not apply to non-vendors. If you've had a bad experience with a technology, you should feel free to express that without facing a burden of proof.
+* Guideline: vendors should generally avoid promoting their own product unless there is a very clear use-case. If someone asks about tools to solve a certain problem, a vendor could say “the big three service providers are alpha, beta, and gamma — i I work at gamma so i’m I'm biased toward that one. If you want to discuss more about the details, direct message me and that would be fine. Constantly posting "you should check out gamma" is not.
```
Comment (emilieschario) [inline content/community.md]:
```suggestion
* Guideline: vendors should generally avoid promoting their own product unless there is a very clear use-case. If someone asks about tools to solve a certain problem, a vendor could say “the big three service providers are alpha, beta, and gamma — I work at gamma so I'm biased toward that one. If you want to discuss more about the details, direct message me," and that would be fine. Constantly posting "you should check out gamma" is not.
```

Comment (mikekaminsky) [reply content/community.md]: @emilieschario thank you!

**Pull request merged**